### ISSUE TITLE
activate lr_scheduler after epoch 10

### ISSUE THREAD

**Issue by Bajo1994** [PyTorchLightning/pytorch-lightning#5632]
> Is there any way to activate lr_scheduler after epoch 10? 

```
{
'scheduler': lr_scheduler, # The LR scheduler instance (required)
'interval': 'epoch', # The unit of the scheduler's step size
'frequency': 1, # The frequency of the scheduler
'reduce_on_plateau': False, # For ReduceLROnPlateau scheduler
'monitor': 'val_loss', # Metric for ReduceLROnPlateau to monitor
'strict': True, # Whether to crash the training if `monitor` is not found
'name': None, # Custom name for LearningRateMonitor to use
}

```

**Comment by awaelchli** [comment]
You could provide a custom scheduler that does nothing until a certain step or epoch is reached. A simple counter variable would do.

**Comment by Bajo1994** [comment]
> @awaelchli Below is the way that I wrote the scheduler. Can you do me a favor and show me how I should add the counter variable to the configure_optimizer?

```
def configure_optimizers(self):
    disOptim=optim.Adam(filter(lambda p:p.requires_grad,self.D.parameters()),lr=self.hparams.lr_dis,betas=(0.0,0.9))
    genOptim=optim.Adam(itertools.chain(self.G.parameters(),self.E.parameters()),lr=self.hparams.lr_gen,betas=(0.0,0.9))
    scheduler_d=optim.lr_scheduler.ExponentialLR(disOptim,gamma=0.99)
    scheduler_g=optim.lr_scheduler.ExponentialLR(genOptim,gamma=0.99)
    return ({'optimizer':disOptim,'frequency':self.hparams.n_critic,'lr_scheduler':{'scheduler':scheduler_d,'interval':'epoch','frequency':1}},
            {'optimizer':genOptim,'frequency':1,'lr_scheduler':{'scheduler':scheduler_g,'interval':'epoch','frequency':1}}) 
```

**Comment by akihironitta** [comment]
You can define a custom lr scheduler like we have a similar one in Bolts. I hope this helps. https://github.com/PyTorchLightning/PyTorch-Lightning-Bolts/blob/master/pl_bolts/optimizers/lr_scheduler.py#L10-L120

**Comment by Bajo1994** [comment]
Many thanks for your great help!
